### PR TITLE
Prepare new release

### LINKED ISSUE #11: detail information for missing element linting rules is not visible
Hi,

we found an error with handling linting errors for missing elements. I.e. 
https://github.com/bpmn-io/bpmnlint/tree/master/docs/rules -> bpmnlint/docs/rules/start-event-required.md 

The situation is, that we have 2 participants in a model. The first one is a simple process with a StartEvent. The second one has a LaneSet and no StartEvent in any lane. Yes, this is an error. The BPMNLinter-Plugin counts this error, but the red X is missing. Thats why the detail information of the error is missing too. The only information is "1 Errors". It took a while to get the details, by adding a breakpoint in the script where the errors are counted.
The example in the docs show this red X, but the process is not placed in a LaneSet.

Maybe you could change the handling of errors a bit. For example add the details to the counter, maybe only if it depends on a missing element. Placing the details at the first element in the process could be a bit hard to realise, because you have to "know" what element is the first... :)

Best regards
Frank

### PR DESCRIPTION
**Pull request by MaxTru**
closes #11

### CODE REVIEW

**Comment by philippfromme** [comment]
One minor thing: We usually use drop shadows for our Screely screenshots.

**Comment by MaxTru** [comment]
> One minor thing: We usually use drop shadows for our Screely screenshots.

Corrected :+1:

**Comment by MaxTru** [comment]
> One minor thing: We usually use drop shadows for our Screely screenshots.

Corrected :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,10 @@ All notable changes to [camunda-modeler-linter-plugin](https://github.com/bpmn-i
 
 ___Note:__ Yet to be released changes appear here._
 
-* `CHORE`: update to `bpmnlint@7.1.0`
+## 0.6.0
+
+* `CHORE`: update to `bpmnlint@7.2.0`
+* `CHORE`: update to `bpmnlint-plugin-camunda@0.5.0`
```
Comment (philippfromme) [inline CHANGELOG.md:12]:
```suggestion
* `CHORE`: update to `bpmn-js-bpmnlint@0.17.0`
```

Comment (MaxTru) [reply CHANGELOG.md:12]: Thanks, done

**Review by philippfromme — APPROVED** [review]
👏🏻

**Pull request merged**